Change the default deletion policy to "permanent", since the recycle bin
doesn't work in a Docker container.
--- a/FreeFileSync/Source/base/structures.h
+++ b/FreeFileSync/Source/base/structures.h
@@ -203,7 +203,7 @@
     //sync direction settings
     SyncDirectionConfig directionCfg = getDefaultSyncCfg(SyncVariant::twoWay);
 
-    DeletionVariant deletionVariant = DeletionVariant::recycler; //use Recycle Bin, delete permanently or move to user-defined location
+    DeletionVariant deletionVariant = DeletionVariant::permanent; //use Recycle Bin, delete permanently or move to user-defined location
 
     //versioning options
     Zstring versioningFolderPhrase;
